reward/reward_distributor.go

```diff
@@ -51,28 +53,41 @@ func NewRewardDistributor(gh governanceHelper) *RewardDistributor {
 }
 
 // getTotalTxFee returns the total transaction gas fee of the block.
-func (rd *RewardDistributor) getTotalTxFee(header *types.Header, rewardConfig *rewardConfig) *big.Int {
+func (rd *RewardDistributor) getTotalTxFee(header *types.Header, rewardConfig *rewardConfig, isKIP71Forked bool) (*big.Int, error) {
 	totalGasUsed := big.NewInt(0).SetUint64(header.GasUsed)
-	totalTxFee := totalGasUsed.Mul(totalGasUsed, rewardConfig.unitPrice)
-	return totalTxFee
+	var totalTxFee *big.Int
+	if isKIP71Forked {
+		if header.BaseFee == nil {
+			logger.Error("KIP-71 hard forked block should have baseFee", "blockNum", header.Number.Uint64())
+			return nil, errInvalidKIP71Block
```

Comment:
If `errInvalidKIP71Block` is not used elsewhere, how about this (typo fixed):
```go
return nil, errors.New("Invalid KIP-71 block with no baseFee")
```

Reply:
Thanks for your proposal.

Reply:
Please resolve this review